test/tests/parallel/set1/spam_virus/spam_junk_folder.bats

```diff
@@ -95,7 +95,7 @@ function teardown() { _default_teardown ; }
 function _should_send_spam_message() {
   _wait_for_smtp_port_in_container
   _wait_for_tcp_port_in_container 10024 # port 10024 is for Amavis
-  _send_email --from 'spam@external.tld' --data 'amavis/spam'
+  _send_email --from 'spam@external.tld' --data 'amavis/spam.txt'
```

Comment:
Like here:

```bash
function send_spam() {
  _send_email --from 'spam@external.tld' --data "${GTUBE}"
}
```

```suggestion
  _send_spam
```

---

Maybe for a future follow-up PR, since I had an approach in mind before the revised `sending.bash` PRs appeared 😝

Reply:
👍🏼 Future PR please :D